### PR TITLE
Add Atoum tests documentation

### PR DESCRIPTION
**Pull request by cconard96**
Waiting for the Jamf plugin to add and push tests to GitHub since it is referenced in the new docs.

### CODE REVIEW

File: source/plugins/atoum.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+Atoum Unit Testing
+-----------------
+
+Goals
+^^^^^
+
+As a plugin's complexity increases so does the possibility of a feature or bug fix breaking some other part of the plugin. For this, it is recommended that plugins have some unit tests in place to detect when expected functionality breaks.
+
+Composer
```
Comment (trasher) [inline source/plugins/atoum.rst:9]: Seems useless since GLPI must be present and already ships atoum.

File: source/plugins/atoum.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            "atoum/atoum": "^3.4"
+        }
+    }
+
+When a developer or contributor installs the plugin they need to navigate to the plugin's directory and run "composer install" to install Atoum.
+
+Bootstrap
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Next, you need to create a bootstrap file to prepare the testing environment. This file should be located at "tests/bootstrap.php".
```
Comment (trasher) [inline source/plugins/atoum.rst:28]:
```suggestion
Next, you need to create a bootstrap file to prepare the testing environment. This file should be located at ``tests/bootstrap.php``.
```

File: source/plugins/atoum.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    if (!$plugin->isActivated('myplugin')) {
+        $plugin->activate($plugin->getID());
+    }
+
+You must replace "myplugin" with the directory name of your plugin.
+
+Unit test files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All unit tests must be placed inside the "tests/units" directory in your plugin.
```
Comment (trasher) [inline source/plugins/atoum.rst:65]:
```suggestion
All unit tests must be placed inside the ``tests/units`` directory in your plugin.
```

File: source/plugins/atoum.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+Each test file has to correspond to an existing class name. If your plugin has a file "inc/test.class.php" with the class name "PluginMypluginTest", the test file must be named "PluginMypluginTest.php".
```
Comment (trasher) [inline source/plugins/atoum.rst:66]:
```suggestion
Each test file has to correspond to an existing class name. If your plugin has a file ``inc/test.class.php`` with the class name ``PluginMypluginTest``, the test file must be named ``PluginMypluginTest.php``.
```

File: source/plugins/atoum.rst [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+Real examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following plugins are a good example of how to implement Atoum tests:
+
+- `JAMF Plugin for GLPI`_
+- `Fields`_
+
+.. _JAMF Plugin for GLPI: https://github.com/cconard96/jamf
+.. _Fields: https://github.com/pluginsGLPI/fields
```
Comment (trasher) [inline source/plugins/atoum.rst:77]:
```suggestion
- `JAMF Plugin for GLPI <https://github.com/cconard96/jamf>`_
- `Fields Plugin for GLPI<https://github.com/pluginsGLPI/fields>`_

```

File: source/plugins/atoum.rst [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+Further reading
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The `Atoum documentation`_ is a good place to start if you are not familiar with unit testing or Atoum.
+
+.. _Atoum documentation: https://atoum.readthedocs.io/en/latest/
```
Comment (trasher) [inline source/plugins/atoum.rst:84]:
```suggestion
The `Atoum documentation <https://atoum.readthedocs.io/>`_ is a good place to start if you are not familiar with unit testing or Atoum.
```

File: source/plugins/index.rst [hunk 7]
```diff
@@ -25,3 +25,4 @@ If you want to see more advanced examples of what it is possible to do with plug
    massiveactions
    tips
    notifications
+   atoum
```
Comment (trasher) [inline source/plugins/index.rst:28]: Should mabye renamed to "test" rather than "atoum".

File: source/plugins/atoum.rst (hunk 1 above)
Comment (cconard96) [reply source/plugins/atoum.rst:9]: That wasn't working before... Seems good now.

**Comment by cconard96** [comment]
Jamf plugin tests are ready. Good for review/merge.

**Pull request merged**